src/plugin.scss

```diff
@@ -20,14 +20,7 @@
   background:rgba(0, 0, 0, .5);
 }
 .vjs-error .vjs-error-display .vjs-modal-dialog-content {
-  font-size: 14px;
-}
-.vjs-close-button.vjs-control.vjs-button {
-  top: 0px;
-  right : -3px;
-}
-.vjs-modal-dialog-content {
-  line-height:1.25;
+  font-size: 12px;
```

Comment:
The font size was changed just to alter the visual appearance. I'll remove the sizing.